src/virtual-machine/compiler/instructions/block.ts

```diff
@@ -15,15 +15,17 @@ export class BlockInstruction extends Instruction {
     this.frame = [...frame]
   }
 }
-
-export class ExitBlockInstruction extends Instruction {
-  constructor() {
-    super('EXIT_BLOCK')
+export class FuncBlockInstruction extends BlockInstruction {
+  args: number
+  constructor(args: number) {
+    super(false)
+    this.tag = 'FUNC_BLOCK'
+    this.args = args
   }
```

Comment:
We should standardize and stick to this style for future classes (where applicable) to reduce duplication:

```typescript
constructor(public args: number) {
  super(false)
  this.tag = 'FUNC_BLOCK'
}
```